src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -28,7 +28,7 @@ public UserController(UserService userService, UserMapper userMapper) {
         this.userMapper = userMapper;
     }
 
-    @PostMapping
+    @PostMapping("/register")
     @ResponseStatus(HttpStatus.CREATED)
     public UserResponseDto create(@RequestBody UserRequestDto userRequestDto) {
```

Comment:
do you already have "/register" in AuthenticationController?

Reply:
I take off this metod (I can1t create user without authentication